.swiftlint.yml

```diff
@@ -0,0 +1,155 @@
+    name: "Return Whitespace"
+    message: "Put an empty line before the return line if two or more other lines in closure."
+    severity: warning
+  single_line_guard:
+    included: ".*.swift"
+    regex: 'guard[^\{]{2,100}else\s*\{\s*\n\s*return.{2,60}\}'
+    name: "Single Line Guard"
+    message: "Use a single line guard for simple checks."
+    severity: warning
+  space_before_comment:
+    included: ".*.swift"
+    regex: '\n[ ]{0}[^\/ \n][^\n]*\n[ ]{0}\/\/|\n[ ]{4}[^\/ \n][^\n]*\n[ ]{4}\/\/|\n[ ]{8}[^\/ \n][^\n]*\n[ ]{8}\/\/|\n[ ]{16}[^\/ \n][^\n]*\n[ ]{16}\/\/|\n[ ]{20}[^\/ \n][^\n]*\n[ ]{20}\/\/|\n[ ]{24}[^\/ \n][^\n]*\n[ ]{24}\/\/'
+    name: "Space Before Comment"
+    message: "Always put a space line before a comment line and always comment code beforehand."
+    severity: warning
+  too_much_indentation:
+    included: ".*.swift"
+    regex: '\n {0}[^\s\/][^\n]*[^,|&]\n+ {5,}\S|\n {4}[^\s\/][^\n]*[^,|&]\n+ {9,}\S|\n {8}[^\s\/][^\n]*[^,|&]\n+ {13,}\S|\n {12}[^\s\/][^\n]*[^,|&]\n+ {17,}\S|\n {16}[^\s\/][^\n]*[^,|&]\n+ {21,}\S|\n {20}[^\s\/][^\n]*[^,|&]\n+ {25,}\S'
+    name: "Too Much Indentation"
+    message: "Don't indent code by more than 4 whitespaces."
+    severity: warning
+  too_much_unindentation:
+    included: ".*.swift"
+    regex: ' {28}\S(.|[^\n]*[^\)][^\ ][^\}])\n+ {0,23}[^\s\/]| {24}\S(.|[^\n]*[^\)][^\ ][^\}])\n+ {0,19}[^\s\/]| {20}\S(.|[^\n]*[^\)][^\ ][^\}])\n+ {0,15}[^\s\/]| {16}\S(.|[^\n]*[^\)][^\ ][^\}])\n+ {0,11}[^\s\/]| {12}\S(.|[^\n]*[^\)][^\ ][^\}])\n+ {0,7}[^\s\/]| {8}\S(.|[^\n]*[^\)][^\ ][^\}])\n+ {0,3}[^\s\/]'
+    name: "Too Much Unindentation"
+    message: "Don't unindent code by more than 4 whitespaces."
+    severity: warning
+  unnecessary_case_break:
+    included: ".*.swift"
+    regex: '(case |default)(?:[^\n\}]+\n){2,}\s*break|\n *\n *break(?:\n *\n|\n *\})'
+    name: "Unnecessary Case Break"
+    message: "Don't use break in switch cases – Swift breaks by default."
+    severity: warning
+  unnecessary_nil_assignment:
+    included: ".*.swift"
+    regex: 'var \S+\s*:\s*[^\s]+\?\s*=\s*nil'
+    name: "Unnecessary Nil Assignment"
+    message: "Don't assign nil as a value when defining an optional type – it's nil by default."
+    severity: warning
+  vertical_whitespace_between_cases:
+    included: ".*.swift"
+    regex: '[^\n{][ \t]*\n[ \t]*(?:case[^\n]+|default):[ \t]*\n'
+    name: "Vertical Whitespace Between Cases"
+    message: "Include a vertical whitespace (empty line) between cases in switch statements."
+    severity: warning
+  vertical_whitespaces_around_mark:
+    included: ".*.swift"
+    regex: '\/\/\s*MARK:[^\n]*(\n\n)|(\n\n\n)[ \t]*\/\/\s*MARK:|[^\s{]\n[^\n\/]*\/\/\s*MARK:'
+    name: "Vertical Whitespaces Around MARK:"
+    message: "Include a single vertical whitespace (empty line) before and none after MARK: comments."
```

Comment:
👎 

I think we should stay with an empty line after `MARK: ` comments. It makes it clearer that the mark delineates a section instead of applying to one specific item.